oat-sa/extension-tao-booklet
# Feature/ADF-1245/Add ES5 bundles

### Related to: https://oat-sa.atlassian.net/browse/ADF-1245

### Requires
- [x] https://github.com/oat-sa/tao-core/pull/3655
- [x] https://github.com/oat-sa/extension-tao-item/pull/600
- [x] https://github.com/oat-sa/extension-tao-itemqti/pull/2260
- [x] https://github.com/oat-sa/extension-tao-qtiprint/pull/97

### Summary

Add bundle tasks for generating an ES5 flavour of some bundles (taoBooklet)
Change the `render` page for using the ES5-flavoured bundles. Use the bundles from each extension instead of the previously made standalone bundle as it was introducing too much coupling (changes in dependencies requested a change in this extension as well otherwise it would not be reflected).

### Details

The internal compiler has been updated, see the companion PR.
We can now have ES5 bundles aside from the regular ones. To properly use them, we need the full chain to declare dependencies to ES5. See the companion PRs for some examples.

### How to test
- checkout the branch
- install the dependencies: `cd tao/views/build && npm i`
- run bundle tasks: `cd tao/views/build && npx grunt taobookletbundle`
- check the booklets generate well both in preview and in PDF mode
